package.json

```diff
@@ -16,6 +16,7 @@
     "@types/node": "^12.7.4",
     "@types/request": "^2.48.2",
     "apollo-errors": "^1.9.0",
+    "apollo-server-express": "^2.9.3",
```

Comment:
I don't think we need this unless we decide to refactor a lot inside server.ts and get rid of our express-graphql server